src/utils/waiting/waiting.go

```diff
@@ -0,0 +1,86 @@
+package waiting
```

Comment:
I must say that i don't really understand what this package gives us. It sounds like all we need one function and transformed it to object that still only one function from it is used? maybe i miss the point?